tests/regression_tests/mgxs_library_no_nuclides/test.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, *args, **kwargs):
         self.mgxs_lib.by_nuclide = False
 
         # Test all MGXS types
-        self.mgxs_lib.mgxs_types = openmc.mgxs.MGXS_TYPES + \
+        self.mgxs_lib.mgxs_types = openmc.mgxs.MGXS_TYPES[:-1] + \
```

Comment:
You could leave currents in that one too ?
Or is there an issue with the output format ? (dimensions of array?)

Reply:
The problem here is that the domain type imposed is 'material', but current can only work on a 'mesh' domain.

Reply:
Ok it's fine to drop that one.

Reply:
get_xs() isn't tested anywhere for any of the MGXS types. All the tests rely on get_mgxs() from the library class. So two options:
1) I don't test get_xs() and remain consistent with the other classes
2) I add a test for get_xs() in every test since it was missing

Reply:
get_xs isnt tested explicitly, but it was being called by one of the tests for every other MGXS types as you can see on coveralls. 
Now that you have changed the condense and hdf5 tests to allow the current MGXS, it is also tested for MeshSurfaceMGXS
Option 2, a unit test for get_xs() would be better. As you want, what you did is good for this PR